docs/howto/queryable-encryption.rst

```diff
@@ -0,0 +1,247 @@
+                        "aws": {
+                            "accessKeyId": "your-access-key-id",
+                            "secretAccessKey": "your-secret-access-key",
+                        }
+                    },
+                )
+            },
+            db_name="encrypted",
+        ),
+    }
+
+    DATABASES["encrypted"]["KMS_CREDENTIALS"] = {
+        "aws": {
+            "key": os.getenv("AWS_KEY_ARN", ""),
+            "region": os.getenv("AWS_KEY_REGION", ""),
+        },
+    }
+
+
+    class EncryptedRouter:
+        # ...
+        def kms_provider(self, model, **hints):
+            return "aws"
+
+
+Configuring the ``encrypted_fields_map``
+========================================
+
+When you :ref:`configure an encrypted database connection
+<qe-configuring-databases-setting>` without specifying an
+``encrypted_fields_map`` in
+:class:`pymongo.encryption_options.AutoEncryptionOpts`, Django MongoDB Backend
+will create an encrypted fields map for you (when ``python manage.py migrate``
+is run), including new data keys, and use it to create collections for models
+with encrypted fields.
+
+The data keys are stored in the key vault :ref:`specified in the Django
+settings <qe-configuring-kms>`. You can view the encrypted fields map by running
+the :djadmin:`showencryptedfieldsmap` command.
+
+To see the keys created by Django MongoDB Backend in the above scenario, you can
+run the following command::
+
+    $ python manage.py showencryptedfieldsmap --database encrypted
+
+You can then use the output of the :djadmin:`showencryptedfieldsmap` command
+to set the ``encrypted_fields_map`` in
+:class:`pymongo.encryption_options.AutoEncryptionOpts` in your Django settings
+if you want to use a pre-defined encrypted fields map in the client instead of
+letting Django MongoDB Backend create them for you.
+
+.. try to explain the chicken/egg scenario here
+
+Of course, if you do this after Django MongoDB Backend has already created the
+collections, you will need to drop the collections first before using the
+pre-defined encrypted fields map.
+
+If you do not want to use the data keys created by Django MongoDB Backend (when
+``python manage.py migrate`` is run), you can generate new data keys with::
+
+    $ python manage.py showencryptedfieldsmap --database encrypted \
+        --create-data-keys
+
+In this scenario, Django MongoDB Backend will use the newly created data keys
+to create collections for models with encrypted fields.
+
+Configuring the Automatic Encryption Shared Library
+===================================================
+
+The :ref:`manual:qe-reference-shared-library` is a preferred alternative to
+:ref:`manual:qe-mongocryptd` and does not require you to start another process
+to perform automatic encryption.
+
+In practice, if you use Atlas or Enterprise MongoDB, ``mongocryptd`` is already
+configured for you, however in such cases the shared library is still
+recommended for use with Queryable Encryption.
```

Comment:
What does "configured for you" mean?

The only "no extra work required" advantage to `mongocryptd` which I can see is that it ships with server binaries, but that's irrelevant for Atlas, and for EA it only matters if the client is running on the same host as the MongoDB server